compaction/compaction.cc

```diff
@@ -455,6 +455,7 @@ class compaction {
     uint64_t _estimated_partitions = 0;
     double _estimated_droppable_tombstone_ratio = 0;
     uint64_t _bloom_filter_checks = 0;
+    std::unordered_map<int32_t, int64_t> _rows_merged;
```

Comment:
This is very opaque. Since it's a histogram, name it as such. Explain what the key is, since int32 doesn't say much.

It's also going to have a measurable impact on compaction speed. Better use one of our histogram types.